examples/use-partial-resource.yaml

```diff
@@ -0,0 +1,74 @@
+# Example: Use a portion of a stock resource
+
+'@context':
+  - https://git.io/vf-examples-jsonld-context
+  - alice: https://alice.example/
+    carpenters: https://carpenters.example/
+
+'@id': rgh:valueflows/valueflows/master/examples/use-partial-resource.yaml
+'@graph':
+
+
+  # tools used by the group of carpenters
+
+  - '@id': carpenters:9f112621-3f13-4f48-a3e2-fd2ca99493ca
+    '@type': EconomicResource
+    resourceClassifiedAs: https://www.wikidata.org/wiki/Q58964 # stock drills
+    currentQuantity:
+      qudt:unit: unit:Number
+      qudt:numericValue: 10
+
+  - '@id': carpenters:8791913c-0152-4c32-bb3f-f25d61df736b
+    '@type': EconomicResource
+    resourceClassifiedAs: https://www.wikidata.org/wiki/Q2365675 # a table saw
+    trackingIdentifier: 38922-44
+    currentQuantity:
+      qudt:unit: unit:Number
+      qudt:numericValue: 1
+
+  # the carpenters are working on a process
+
+  - '@id': carpenters:e18c808c-929a-450d-9d0d-3b3f531bc125
+    '@type': Process
+    name: Make a small shed
+
+  - '@id': carpenters:e72f7d1d-4f0a-4518-a44c-96658cafda94
+    '@type': EconomicEvent
+    inputOf: carpenters:e18c808c-929a-450d-9d0d-3b3f531bc125 # the process
+    action: use
+    provider: https://carpenters.example/
+    receiver: https://carpenters.example/
+    affects: carpenters:9f112621-3f13-4f48-a3e2-fd2ca99493ca # drills
+    observedQuantity: # hours of use of the 3 drills
+      qudt:unit: unit:Hour
+      qudt:numericValue: 3.5
+    usedQuantity: # portion of the stock resource that is used
+      qudt:unit: unit:Number
+      qudt:numericValue: 3
+    observedTime: # the 3 drills are unavailable during this time
+      '@type': time:ProperInterval
+      time:intervalStarts:
+        '@type': time:Instant
+        time:inXSDDateTimeStamp: 2018-10-20T08:00:00-5:00
+      time:intervalEnds:
+        '@type': time:Instant
+        time:inXSDDateTimeStamp: 2018-10-20T11:30:00-5:00
+
+  - '@id': carpenters:f325a950-5737-488d-b122-8f21114d0eb9
+    '@type': EconomicEvent
+    inputOf: carpenters:e18c808c-929a-450d-9d0d-3b3f531bc125 # the process
+    action: use
+    provider: https://carpenters.example/
+    receiver: https://carpenters.example/
+    affects: carpenters:carpenters:8791913c-0152-4c32-bb3f-f25d61df736b # table saw
+    observedQuantity: # hours of use of the table saw (no need for usedQuantity)
```

Comment:
Keeping in mind [Open-world assumption](https://en.wikipedia.org/wiki/Open-world_assumption) one would need to consider the quantity `unknown`, since we need a way to provide that quantity, we should make it required and mention that one should not make assumptions based on lack of information.

Reply:
I think the quantity of the transferable resource (not the time-based use quantity) can be considered to be that of the resource being used.  Doesn't work?

Reply:
How one can tell difference between 'missing some information' and 'just use the current quantity of the resource', it also would depend on having access to the information about current quantity of the resource at the time of event. I think event has to capture that information and provide it as part of asserted knowledge rather than leaving it ambiguous.

Reply:
But the observed quantity is the only one that is used for anything in terms of counting / accounting, I'm proposing.  The observed quantity is the one that people need to observe.  That's why I'm trying to get that information into one property.  Other quantities are optional.  So its fine for usedQuantity to be unknown imo.

Reply:
What available quantity of the resource one would calculate at `2018-10-20T09:00:00-5:00` ?

Reply:
2.5